drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml

```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    configuration:
```

Comment:
configuration:   needs to be changed to   config: 


@ctowns Can you help with this please

Reply:
Done!